README.md

```diff
@@ -1 +1,156 @@
-# nexus-repository-apk
+<!--
+
+    Sonatype Nexus (TM) Open Source Version
+    Copyright (c) 2019-present Sonatype, Inc.
+    All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+
+    This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+    which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+
+    Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+    of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+    Eclipse Foundation. All other trademarks are the property of their respective owners.
+
+-->
+# Nexus Repository apk Format
```

Comment:
NOTE: It is weird to me apk is lower case here and upper below (header).  Maybe context I am missing (thus note).

Reply:
@joedragons Good catch, i am looking at my bookmarks and i made a note of this as well. In the [official documentation](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management) they have `apk is the tool used to install, upgrade, or delete software on a running system.` -- This to me reads all instances of apk should be lower-case. Does this sound right to you too?

Reply:
@allenhsieh agree.  npm is the same (and we struggle with that consistency too, if it makes you feel better;D).

Reply:
@joedragons haha oh boy, good to know others have trouble with this.

in: https://github.com/sonatype-nexus-community/nexus-repository-apk/pull/1/commits/855e16045f8ba3f55734137b4ad31b275d99754a